xrpl/asyncio/clients/exceptions.py

```diff
@@ -17,7 +17,7 @@ def __init__(self: XRPLRequestFailureException, result: Dict[str, Any]) -> None:
             result: the error result returned by the ledger.
         """
         self.error = result["error"]
-        if "error_message" in result:
+        if "error_message" in result and result["error_message"] is not None:
             self.error_message = result["error_message"]
         elif "error_exception" in result:
```

Comment:
should we do the same here?

Reply:
It doesn't matter there if it's `None`, especially with the intialization I just added - if it's `None` the value just doesn't change in effect